samples/contrib/aws-samples/simple_train_pipeline/README.md

```diff
@@ -41,57 +41,91 @@ An example pipeline with only [train component](https://github.com/kubeflow/pipe
    boto3.resource('s3').Bucket(bucket).Object(data_key).upload_fileobj(buf)
    ```
    Run this file `python s3_sample_data_creator.py`
-3. Prepare an IAM role with permissions to run SageMaker jobs and access to S3 buckets.   
-   
-   create a new file "trust.json" with following content
-   ```buildoutcfg 
-   {
-     "Version": "2012-10-17",
-     "Statement": [
-       {
-         "Sid": "",
-         "Effect": "Allow",
-         "Principal": {
-           "Service": "sagemaker.amazonaws.com"
-         },
-         "Action": "sts:AssumeRole"
-       }
-     ]
-   }
-   ```
-   ```buildoutcfg
-
-   # run these commands to create a role named "SageMakerExecutorKFP" with SageMaker and S3 access
-   aws iam create-role --role-name SageMakerExecutorKFP --assume-role-policy-document file://trust.json
-   aws iam attach-role-policy --policy-arn arn:aws:iam::aws:policy/AmazonSageMakerFullAccess --role-name SageMakerExecutorKFP
-   aws iam attach-role-policy --policy-arn arn:aws:iam::aws:policy/AmazonS3FullAccess --role-name SageMakerExecutorKFP
-   
-   # Note down the role ARN
-   aws iam get-role --role-name SageMakerExecutorKFP     # | jq .Role.Arn
-   ```
-4. Add 'aws-secret' to your Kubeflow namespace.
-   ```
-   # 1. get aws key and secret in base64 format: 
-
-   echo -n "<AWS_ACCESS_KEY_ID>" | base64
-   echo -n "<AWS_SECRET_ACCESS_KEY>" | base64
-
-   # 2. Create new file secret.yaml with following content
-   
-   apiVersion: v1
-   kind: Secret
-   metadata:
-     name: aws-secret
-     namespace: kubeflow
-   type: Opaque
-   data:
-     AWS_ACCESS_KEY_ID: <base64_AWS_ACCESS_KEY_ID>
-     AWS_SECRET_ACCESS_KEY: <base64_AWS_SECRET_ACCESS_KEY>
-     
-   # 3. Now apply to the cluster's kubeflow namespace:
- 
-   kubectl -n kubeflow apply -f secret.yaml 
-   ```
+3. **Prepare IAM Roles**
```

Comment:
I think this is one time setup so I would suggest keep it at inside `aws-samples` or point to any README you may have in component folder so that we don't need to replicate same instruction everywhere.

Reply:
done